aidants_connect_web/views/espace_aidant.py

```diff
@@ -61,3 +68,52 @@ def validate_cgus(request):
             "form": form,
         },
     )
+
+
+@login_required
+@activity_required
+@user_is_aidant
+@require_http_methods(["GET", "POST"])
+def switch_main_organisation(request: HttpRequest):
+    aidant: Aidant = request.user
+
+    if request.method == "GET":
+        return render(
+            request,
+            "aidants_connect_web/espace_aidant/switch_main_organisation.html",
+            {
+                "aidant": aidant,
+                "next_url": request.GET.get("next", ""),
+                "organisations": aidant.organisations,
+                "disable_change_organisation": True,
+            },
+        )
+
+    organisation_id = request.POST["organisation"]
+
+    try:
+        organisation = Organisation.objects.get(pk=organisation_id)
+    except Organisation.DoesNotExist:
+        django_messages.error(
+            request,
+            f"Aucune organisation portant l'identifiant {organisation_id} n'existe.",
+        )
+        return redirect("espace_aidant_switch_main_organisation")
+
+    if organisation.id not in Organisation.objects.filter(
+        aidants__pk=aidant.pk
+    ).values_list("pk", flat=True):
```

Comment:
Un simple `if organisation not in aidant.organisations` serait plus lisible, et éviterait une requête supplémentaire.

Reply:
Pourquoi n'y ai-je pas pensé !?

Reply:
C'est l'effet "tête dans le guidon", pas d'inquiétude, les PR c'est fait pour ça. :)

Reply:
Ah oui. Je me souviens pourquoi j'y ai pas pensé. Si j'essaie ça, j'ai une erreur `argument of type 'ManyRelatedManager' is not iterable`